mne/viz/topo.py

```diff
@@ -70,6 +70,8 @@ def iter_topography(info, layout=None, on_pick=None, fig=None,
         The current axis of the topo plot.
     ch_dx : int
         The related channel index.
+    name : str
```

Comment:
is this how channel names are called everywhere else? Isn't it `ch_name` or something like that?